src/goose.rs

```diff
@@ -854,9 +863,9 @@ impl GooseUser {
     ///       let _goose = user.get("/path/to/foo/").await;
     ///     }
     /// ```
-    pub async fn get(&self, path: &str) -> GooseResponse {
+    pub async fn get(&self, path: &str) -> Result<GooseResponse, mpsc::error::SendError<bool>> {
```

Comment:
Are you sure that's the only error that can happen?

Can be follow-up though as it's internal.

Reply:
At this time it is the only error that can happen, so there was no need to make it generic or add a custom wrapper. We can tackle this when it's necessary/useful.